spec/spec_helper_local.rb

```diff
@@ -35,5 +35,5 @@
     }
   end
 
-  it { is_expected.to compile.with_all_deps }
+  # it { is_expected.to compile.with_all_deps }
```

Comment:
Is there any reason this is commented out?

Reply:
It didn't work in combination with rspec-puppet-facts and also felt redundant. All the mod files should already include `it { is_expected.to compile.with_all_deps }`